crates/autopilot/src/lib.rs

```diff
@@ -541,10 +541,6 @@ pub async fn main(args: arguments::Arguments) {
         .spawn();
     }
 
-    tokio::select! {
-        result = serve_metrics => panic!("serve_metrics exited {:?}", result),
-        result = db_metrics => panic!("db_metrics exited {:?}", result),
-        result = maintenance_task => panic!("maintenance exited {:?}", result),
-        result = auction_transaction => panic!("auction_transaction exited {:?}", result),
-    };
```

Comment:
Isn't this technically a functional change where we no longer panic if certain Tokio tasks exit?